src/resources/espnow.cc

```diff
@@ -173,8 +173,53 @@ static void espnow_recv_cb(const uint8_t *mac_addr, const uint8_t *data, int dat
   }
 }
 
+static wifi_phy_rate_t map_toit_rate_to_esp_idf_rate(int toit_rate) {
```

Comment:
Wouldn't this be a lot smaller as a constant table of 0x2b entries? Not entirely sure why there are holes in there in the first place.